NEWS.md

```diff
@@ -243,6 +243,12 @@ Library improvements
 
     * The `randexp` and `randexp!` functions are exported ([#9144])
 
+  * File
+
+    * Added function `readlink` which returns the value of a symbolic link "path" ([#10714]).
+
+    * The `cp` function now accepts keyword arguments `recursive`, `remove_destination` and `follow_symlinks` ([#10506]).
```

Comment:
Small nitpick, I think this should refer to the PR in which it was implemented (here) rather than the issue it closes?